docs/guide.md

```diff
@@ -275,7 +256,7 @@ type FooBar struct {
 var Set = wire.NewSet(
     ProvideFoo,
     ProvideBar,
-    FooBar{})
+    wire.Struct(new(FooBar), "*"))
```

Comment:
Hm. The `"*"` syntax isn't introduced until later in the prose. Perhaps use:

```suggestion
    wire.Struct(new(FooBar), "Foo", "Bar"))
```

Would be good to change the field names or types to differentiate a little. (Maybe `MyFoo` and `MyBar` for the field names?)

Reply:
Done.